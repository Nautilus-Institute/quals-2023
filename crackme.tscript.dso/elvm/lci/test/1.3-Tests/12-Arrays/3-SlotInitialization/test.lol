HAI 1.3
	I HAS A foo ITZ A BUKKIT
	foo HAS A var1 ITZ WIN
	foo HAS A var2 ITZ FAIL
	foo HAS A var3 ITZ -1
	foo HAS A var4 ITZ 2
	foo HAS A var5 ITZ -3.456
	foo HAS A var6 ITZ 6.789
	foo HAS A var7 ITZ "abc"
	VISIBLE SUM OF 0 AN foo'Z var1
	VISIBLE SUM OF 0 AN foo'Z var2
	VISIBLE foo'Z var3
	VISIBLE foo'Z var4
	VISIBLE foo'Z var5
	VISIBLE foo'Z var6
	VISIBLE foo'Z var7
KTHXBYE
