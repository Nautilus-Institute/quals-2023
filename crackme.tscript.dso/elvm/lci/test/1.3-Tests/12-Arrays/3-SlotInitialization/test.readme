This test is designed to check that slots can be initialized when declared.
