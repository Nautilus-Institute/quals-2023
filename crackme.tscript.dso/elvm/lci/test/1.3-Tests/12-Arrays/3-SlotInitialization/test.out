1
0
-1
2
-3.45
6.78
abc
